Bottom: 95278e62e440233429c82c18ced4e78e1880b2bc
Top:    69502db5be6a1c931e206986cd9b7169ec0d2e48
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-22 17:01:12 -0400

Refresh of phytec

---

diff --git a/sound/soc/fsl/Makefile b/sound/soc/fsl/Makefile
index f406470..5806d11 100644
--- a/sound/soc/fsl/Makefile
+++ b/sound/soc/fsl/Makefile
@@ -17,4 +17,5 @@ obj-$(CONFIG_SND_SOC_MPC5200_AC97) += mpc5200_psc_ac97.o
 
 # MPC5200 Machine Support
 obj-$(CONFIG_SND_MPC52xx_SOC_EFIKA) += efika-audio-fabric.o
+obj-$(CONFIG_SND_MPC52xx_SOC_PCM030) += pcm030-audio-fabric.o
